s3bucket/main.tf

```diff
@@ -0,0 +1,46 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_s3_bucket" "bucket" {
+    bucket = "login-gov-${var.bucket_name}-${var.env_name}-${data.aws_caller_identity.current.account_id}-${var.region}"
+    acl    = "private"
```

Comment:
I think this should probably include an empty `policy { }` section. Based on experience with the prod logs bucket it seems that Terraform won't revert manual changes to the bucket policy unless the section is explicitly set.

Reply:
If I try to set an empty policy I get this error "policy must be a single value, not a list".